climpred/tests/test_comparisons.py

```diff
@@ -146,5 +154,5 @@ def test_m2m(PM_da_ds1d):
     # very weak testing here
     assert eforecast.size == aforecast.size
     assert ereference.size == areference.size
-    #assert_equal(eforecast,aforecast)
-    #assert_equal(ereference,areference)
+    # assert_equal(eforecast,aforecast)
```

Comment:
Should this be removed completely if it's commented out?

Reply:
> If this works on lazy and nonlazy data. Great. Could add a lazy test.

I tested. If you run `.load()` on a non-dask xarray it doesn't break. Just returns the same object.